docs/guide/blog/connect-blockchain.md

```diff
@@ -42,16 +42,16 @@ Your blockchain client has only two dependencies:
 - `ignite` for the `cosmosclient` blockchain client
 
 ```go
-module github.com/cosmonaut/blogclient
+module github.com/username/blogclient
 
-go 1.17
+go 1.18
 
 require (
-	github.com/cosmonaut/blog v0.0.0-00010101000000-000000000000
-	github.com/ignite-hq/cli v0.19.2 
+	github.com/username/blog v0.0.0-00010101000000-000000000000
+	github.com/ignite-hq/cli v0.20.3
```

Comment:
is this the Ignite CLI version? If so, can we test with 0.21 and update all mentions of the version?